manifests/params.pp

```diff
@@ -98,7 +92,7 @@
       }
     }
     'RedHat': {
-      $config_root_ini         = '/etc/php.d'
+      $config_root             = $php::globals::globals_config_root
```

Comment:
You are right. This is my mistake. I'm just going to revert this part and leave it was as this PR is only for Debian/Ubuntu.